next.clayui.com/content/docs/css/components/loading-indicator.md

```diff
@@ -0,0 +1,156 @@
+---
+title: "Loading Indicator"
+description: "The loading indicator shows the user that an external process, like a connection, is being executed."
+---
+
+<div class="nav-toc">
+
+- [Loading Icon Dotted](#loading-icon-dotted)
```

Comment:
These titles no longer need to repeat the name of the component and I think we can better organize the information here, something like this:

```markdown
- [Sizes](#sizes)
- [Variations](#variations)
  - [Positive](#positive)
  - [Negative](#negative)
- [Composable](#composable)
  - [Button](#button)
  - [Autocomplete](#autocomplete)
```

Reply:
Makes sense Matu! Thanks!